conanfile.py

```diff
@@ -317,24 +317,19 @@ def validate(self):
             raise ConanInvalidConfiguration("Only versions 5+ are support")
 
     def requirements(self):
+        for req in self.conan_data["requirements"]:
+            if self.options.internal and "fdm_materials" in req:
+                continue
+            self.requires(req)
+        if self.options.internal:
+            for req in self.conan_data["requirements_internal"]:
+                if "fdm_materials" in req:
+                    continue
```

Comment:
Why would `fdm_materials` be in the internal conan requirements?

Reply:
Otherwise you will endup with a conflict when building with the internal option True